apps/mosaico/src/Main.purs

```diff
@@ -818,6 +818,9 @@ notFoundPage env {params: { path } } = do
     ["rss.xml"] /\ Paper.HBL -> redir "https://lettera.api.ksfmedia.fi/v4/list/frontpage?paper=HBL"
     ["rss.xml"] /\ Paper.VN -> redir "https://lettera.api.ksfmedia.fi/v4/list/frontpage?paper=VN"
     ["rss.xml"] /\ Paper.ON -> redir "https://lettera.api.ksfmedia.fi/v4/list/frontpage?paper=ON"
+    ["bruksvillkor/"] /\ Paper.HBL -> redir "https://www.hbl.fi/sida/bruksvillkor"
```

Comment:
This won't work, it needs to be `["bruksvillkor",""]` (see a few lines above for an example).

Reply:
Thanks. Fixed now.